src/Reporter.cpp

```diff
@@ -422,6 +422,21 @@ namespace geopm
             {"time-hint-unset (s)", {"TIME_HINT_UNSET"}, sample_only},
         };
 
+        auto all_names = m_platform_io.signal_names();
+        std::vector<m_sync_field_s> gpu_sync_fields = {
```

Comment:
```cpp
std::vector<m_sync_field_s> gpu_sync_fields{
```
The code previously called the vector constructor twice, once for constructing the vector in the right of the `=`, once for calling the copy constructor. Instead just call the overloaded vector constructor once.